src/vs/workbench/contrib/welcome/gettingStarted/common/gettingStartedContent.ts

```diff
@@ -130,6 +130,17 @@ export const startEntries: GettingStartedStartEntryContent = [
 			command: 'git.clone',
 		}
 	},
+	{
+		id: 'topLevelGitOpen',
+		title: localize('gettingStarted.topLevelGitOpen.title', "Open Repository..."),
+		description: localize('gettingStarted.topLevelGitOpen.description', "Connect to a remote repository or pull request to browse, search, edit, and commit"),
+		when: 'isWeb && workspaceFolderCount == 0',
```

Comment:
Why `workspaceFolderCount == 0`, rather than just `when: 'isWeb',`?

The other open entries don't seem to depend on what's already open, and I'm thinking it'd be alright if a user wants to open another GH repo even if they had another folder open?

Reply:
Agreed. I also verified that this action works as expected on vs code web and doesn't show for codespaces.